Cargo.toml

```diff
@@ -25,7 +25,7 @@ path = "src/cli/main.rs"
 
 [dependencies]
 revault_tx = { version = "0.2", features = ["use-serde"] }
-revault_net = { git = "https://github.com/revault/revault_net" }
+revault_net = { git = "https://github.com/darosior/revault_net", branch = "always_respond" }
```

Comment:
Since it's merged you can use master now?

Reply:
I just merged it 5 minutes ago..

Reply:
Ah sorry!

Reply:
Done now :)